api/engineapi/data.py

```diff
@@ -244,7 +247,7 @@ def at_least_one_of_contract_id_and_contract_address(cls, values):
 
 class CallRequest(BaseModel):
     id: UUID
-    contract_id: UUID
+    contract_id: UUID = Field(alias="registered_contract_id")
     contract_address: str
```

Comment:
When you create a `CallRequest` object in `orm_mode`, what happens to `contract_address` field? There is no `contract_address` column in table.

Reply:
Since we are using only in one place and manually define `contract_address` it works, but you are right, it is not cleare and I will set this `contract_address: Optional[str] = None`